### PR TITLE
FSET-1311 Changes to send out 2 emails when admin user's email is changed

### PR DESCRIPTION
**Pull request by ian-stainer**
Changes to send out 2 emails (1 to old address and 1 to new address) when an admin user's email address is changed

### CODE REVIEW

File: app/uk/gov/hmrc/fsetemailrenderer/templates/fasttrack/csrEmailChangedSentToNewAddress.scala.html [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+    }
+    <p style="font-size: 19px; line-height: 1.315789474; margin: 0 0 30px 0;">The email address for your account has been changed to <b>@{params.parameters("newEmail")}</b></p>
+    <h2 style="font-size: 32px; line-height: 1.315789474; margin: 0 0 30px 0;">Sign in using this email address and current password</h2>
+    <p style="font-size: 19px; line-height: 1.315789474; margin: 0 0 30px 0;">Your password has not been changed.</p>
+    <p style="font-size: 19px; line-height: 1.315789474; margin: 0 0 30px 0;">Visit the sign in page to continue
+        <br> (<a href="/admin/signin.html">https://admin.tax.service.gov.uk/csr-fast-track-admin/signin</a>)</p>
+
+	<p style="font-size: 19px; line-height: 1.315789474; margin: 0 0 30px 0;">Thank you</p>
+    <p style="font-size: 19px; line-height: 1.315789474; margin: 0 0 30px 0;">Fast Track team</p>
+    <p style="font-size: 19px; line-height: 1.315789474; margin: 0 0 30px 0;">If you need more help, call us on 01276 400 333.</p>
```
Comment (charge-valtech) [inline app/uk/gov/hmrc/fsetemailrenderer/templates/fasttrack/csrEmailChangedSentToNewAddress.scala.html]: I've accidentally given you the wrong phone number here, could it please be updated to 03000 542211

File: app/uk/gov/hmrc/fsetemailrenderer/templates/fasttrack/csrEmailChangedSentToNewAddress.scala.txt [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+Sign in using this email address and current password
+Your password has not been changed.
+
+Visit the sign in page to continue (https://admin.tax.service.gov.uk/csr-fast-track-admin/signin)
+
+Thank you
+
+Fast Track team
+
+If you need more help, call us on 01276 400 333.
```
Comment (charge-valtech) [inline app/uk/gov/hmrc/fsetemailrenderer/templates/fasttrack/csrEmailChangedSentToNewAddress.scala.txt]: I've accidentally given you the wrong phone number here, could it please be updated to 03000 542211

File: app/uk/gov/hmrc/fsetemailrenderer/templates/fasttrack/csrEmailChangedSentToOldAddress.scala.html [hunk 3]
```diff
@@ -4,11 +4,10 @@
         <p style="font-size: 19px; line-height: 1.315789474; margin: 0 0 30px 0;">Dear @{params.parameters("name")}</p>
         <br/>
     }
-    <p style="font-size: 19px; line-height: 1.315789474; margin: 0 0 30px 0;">Your email address has been changed to @{params.parameters("newEmail")}</p>
-    <p style="font-size: 19px; line-height: 1.315789474; margin: 0 0 30px 0;">If this is unexpected please contact us immediately using the details below.</p>
+    <p style="font-size: 19px; line-height: 1.315789474; margin: 0 0 30px 0;">The email address for your account has been changed to <b>@{params.parameters("newEmail")}</b></p>
+    <h2 style="font-size: 32px; line-height: 1.315789474; margin: 0 0 30px 0;">You need to sign in with your new email address</h2>
+    <p style="font-size: 19px; line-height: 1.315789474; margin: 0 0 30px 0;">If you think there's been a mistake, please contact us on 01276 400 333</p>
```
Comment (charge-valtech) [inline app/uk/gov/hmrc/fsetemailrenderer/templates/fasttrack/csrEmailChangedSentToOldAddress.scala.html]: I've accidentally given you the wrong phone number here, could it please be updated to 03000 542211

File: app/uk/gov/hmrc/fsetemailrenderer/templates/fasttrack/csrEmailChangedSentToOldAddress.scala.txt [hunk 4]
```diff
@@ -0,0 +1,10 @@
+@(params: uk.gov.hmrc.fsetemailrenderer.controllers.model.Params)
+@if(params.parameters("name").toString.trim.nonEmpty){Dear @{params.parameters("name")}}
+
+The email address for your account has been changed to @{params.parameters("newEmail")}
+You need to sign in with your new email address
+If you think there's been a mistake, please contact us on 01276 400 333
```
Comment (charge-valtech) [inline app/uk/gov/hmrc/fsetemailrenderer/templates/fasttrack/csrEmailChangedSentToOldAddress.scala.txt]: I've accidentally given you the wrong phone number here, could it please be updated to 03000 542211

File: app/uk/gov/hmrc/fsetemailrenderer/templates/fasttrack/csrEmailChangedSentToNewAddress.scala.html (hunk 1 above)
Comment (ian-stainer) [inline app/uk/gov/hmrc/fsetemailrenderer/templates/fasttrack/csrEmailChangedSentToNewAddress.scala.html]: This has now been updated

File: app/uk/gov/hmrc/fsetemailrenderer/templates/fasttrack/csrEmailChangedSentToNewAddress.scala.txt (hunk 2 above)
Comment (ian-stainer) [inline app/uk/gov/hmrc/fsetemailrenderer/templates/fasttrack/csrEmailChangedSentToNewAddress.scala.txt]: This has now been updated

File: app/uk/gov/hmrc/fsetemailrenderer/templates/fasttrack/csrEmailChangedSentToOldAddress.scala.html (hunk 3 above)
Comment (ian-stainer) [inline app/uk/gov/hmrc/fsetemailrenderer/templates/fasttrack/csrEmailChangedSentToOldAddress.scala.html]: This has now been updated

File: app/uk/gov/hmrc/fsetemailrenderer/templates/fasttrack/csrEmailChangedSentToOldAddress.scala.txt (hunk 4 above)
Comment (ian-stainer) [inline app/uk/gov/hmrc/fsetemailrenderer/templates/fasttrack/csrEmailChangedSentToOldAddress.scala.txt]: This has now been updated

**Comment by kjharkin** [comment]
Looks good

**Pull request merged**